### DISCUSSION TITLE
[강의] 테스트 코드 강의 추천합니다

### DISCUSSION THREAD

**Discussion by nitronium102** [SWM-Space-Odyssey/WeddingMate_BackEnd#23] [Show and tell]
https://www.inflearn.com/course/%EC%8A%A4%ED%94%84%EB%A7%81%EB%B6%80%ED%8A%B8-junit-%ED%85%8C%EC%8A%A4%ED%8A%B8#reviews

짱조음~👍👍👍

**Comment by lina1919** [comment]
오 꼭 들어볼게요~~!
